packages/builder/src/components/base/images/ImageCrop.tsx

```diff
@@ -91,8 +91,8 @@ export default function ImageCrop({
               setCompletedCrop(c);
             }}
             aspect={dimensions.width / dimensions.height}
-            maxHeight={dimensions.height}
-            maxWidth={dimensions.width}
+            maxHeight={undefined}
+            maxWidth={undefined}
```

Comment:
would you mind removing `maxHeight` and `maxWidth` instead of assigning undefined?
Also a quick rebase with main would be nice. Will approve it afterwards.

Thanks for you contrbution!

Reply:
I pushed a new commit that removes these 2 lines altogether